packages/medusa/src/api-v2/admin/product-categories/query-config.ts

```diff
@@ -11,10 +11,39 @@ export const defaults = [
   "updated_at",
   "metadata",
 
+  "*category_children",
+]
+
+export const allowed = [
+  "id",
+  "name",
+  "description",
+  "handle",
+  "is_active",
+  "is_internal",
+  "rank",
+  "parent_category_id",
+  "created_at",
+  "updated_at",
+  "metadata",
+
   "parent_category.id",
   "parent_category.name",
+  "parent_category.handle",
```

Comment:
Should we just do `*parent_category`, or is there an issue with circular dependencies? Same for the children

Reply:
Think we can do that. Will update it 👍